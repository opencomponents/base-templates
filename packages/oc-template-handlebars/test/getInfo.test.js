const { getInfo } = require('../index.js');

const currentVersion = require('../package.json').dependencies.handlebars;

test('should return the correct info', () => {
  const info = getInfo();

  info.externals.forEach(external => {
    if (external.name === 'handlebars')
      external.url = external.url.replace(currentVersion, '6.6.6');
  });
  delete info.version;

  expect(info).toMatchSnapshot();
});
