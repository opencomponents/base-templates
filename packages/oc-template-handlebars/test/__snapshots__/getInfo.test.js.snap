// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct info 1`] = `
Object {
  "externals": Array [
    Object {
      "global": "Handlebars",
      "name": "handlebars",
      "url": "https://unpkg.com/handlebars@6.6.6/dist/handlebars.runtime.min.js",
    },
  ],
  "type": "oc-template-handlebars",
}
`;
